installers/python-ci/radiasoft-download.sh

```diff
@@ -6,9 +6,9 @@ python_ci_main() {
     if [[ ! -r setup.py ]]; then
         install_err 'no setup.py'
     fi
-    local i
-    local p=0
-    local r=$(basename "${GITHUB_REPOSITORY:-MISSING}")
+    declare i
+    declare p=0
+    declare r=$(basename "${GITHUB_REPOSITORY:-MISSING}")
```

Comment:
Why change these from local to declare?

Reply:
https://github.com/radiasoft/discuss/issues/15

Reply:
Resolve privately.